Dockerfile

```diff
@@ -73,13 +74,21 @@ RUN tlmgr update --self && \
     mktextfm larm0600 && \
     mktextfm lati1095 && \
     mktextfm latt1095 && \
-    # clean up unneeded packages
+    # Clean up unneeded packages.
     apk del wget xz tar && \
     rm -rf /var/cache/apk/ && mkdir /var/cache/apk/ && \
-    rm -rf /usr/share/man && \
-    mkdir /workdir
+    rm -rf /usr/share/man
+
+# Install additional packages manually.
+RUN mkdir /tmp/lib
+WORKDIR /tmp/lib
+COPY lib .
+RUN TEXLIB="$(kpsewhich -var-value=TEXMFHOME)/tex/latex" && \
+    mkdir -p $TEXLIB && \
+    cp * $TEXLIB
```

Comment:
Oh boy that's the real magic right there. You barely added 6LOC to this Dockerfile and you've already out-magicked me.